app/controllers/users_controller.rb

```diff
@@ -1,11 +1,11 @@
 class UsersController < ApplicationController
-  def show
-    @user = User.find_by id: params[:id]
-    return if @user
+  include Pagy::Backend
 
-    flash[:danger] = t "users_not_found"
-    redirect_to root_path
-  end
+  before_action :logged_in_user, except: [:show, :new, :create]
+  before_action :correct_user, only: %i[:edit, :update]
+  before_action :find_user, except: [:index, :new, :create]
```

Comment:
```suggestion
  before_action :find_user, except: %i(index new create)
  before_action :correct_user, only: %i(edit update)
```

em dùng %i() hoặc [], ai lại dùng nửa nạc nửa mỡ thế kia